apps/challenges/aws_utils.py

```diff
@@ -952,12 +953,19 @@ def create_eks_cluster(challenge):
             config_text = yaml.dump(cluster_config, default_flow_style=False)
             config_file = NamedTemporaryFile(delete=True)
             config_file.write(config_text.encode())
-            ChallengeEvaluationCluster.objects.create(
-                challenge=challenge_obj,
-                name=cluster_name,
-                cluster_endpoint=cluster_ep,
-                cluster_ssl=cluster_cert,
-            )
+            try:
+                cluster_details =  ChallengeEvaluationCluster.objects.get(
```

Comment:
@Ayukha If this block gets executed, then where are we using the `cluster_details` variable?